app/controllers/ops_controller/settings.rb

```diff
@@ -153,6 +153,7 @@ def region_edit
       @edit[:region].description = @edit[:new][:description]
       begin
         @edit[:region].save!
+        MiqRegion.my_region_clear_cache
```

Comment:
It would be nice to have a presenter object that has the knowledge of the region data presented to the user so it can be determine if this data should be reloaded.  This feels really out of place since the UI code shouldn't know about the cache and it clears the cache even if the change isn't used by the UI (note, in this case, I think description is the only thing in the region table you can change so it doesn't matter).

With that said, I guess this is fine.